scripts/aggregate_claim_metrics.py

```
@@ -0,0 +1,524 @@
+        )
+        return report, 2
+
+    m = report["metrics"][h.upper()]
+    report["headline"] = {
+        "metric": h.upper(),
+        "n": m["n"],
+        "N": report["N_claim"],
+        "rate": m["rate"],
+        "label": m["definition"],
+        "role": m["role"],
+        "diagnostic_only": h == "s3" or diagnostic_only and h != "s1",
```

Ruff (0.15.21)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)